koku/masu/external/accounts/db/cur_accounts_db.py

```diff
@@ -22,7 +22,26 @@
 
 # pylint: disable=too-few-public-methods
 class CURAccountsDB(CURAccountsInterface):
-    """Provider interface defnition."""
+    """Provider interface definition."""
+
+    @staticmethod
+    def get_authentication(provider):
+        """Return either provider_resource_name or credentials."""
+        if provider.authentication.provider_resource_name:
+            return provider.authentication.provider_resource_name
+        elif provider.authentication.credentials:
+            return provider.authentication.credentials
+        return None
+
+    @staticmethod
+    def get_billing_source(provider):
+        """Return either bucket or data_source."""
+        if provider.billing_source:
+            if provider.billing_source.bucket:
```

Comment:
Same as my comment about `get_authentication()` - might be good to have a comment about the deprecation.